packages/jss/src/Jss.js

```diff
@@ -2,9 +2,8 @@
 import isInBrowser from 'is-in-browser'
 import StyleSheet from './StyleSheet'
 import PluginsRegistry from './PluginsRegistry'
-import rulesPlugins from './plugins/rules'
 import sheets from './sheets'
-import StyleRule from './rules/StyleRule'
+import {plugins as defaultPlugins, StyleRule, KeyframesRule} from './plugins/index'
```

Comment:
```suggestion
import {plugins as defaultPlugins, StyleRule, KeyframesRule} from './plugins'
```

Reply:
This is awesome

Reply:
for some reason without index rollup complains and can't build, try yarn build

Reply:
weird. will try it.

Reply:
yeah, rollup has some kind of problem 🤔